content/en/events/upcoming-events/gsoc-2025.md

```diff
@@ -410,187 +248,169 @@ Batch Processing Gateway can help enterprise environments, managing Apache Spark
   - Ensure users can efficiently analyze job execution, detect bottlenecks, and optimize data processing and ML workflows within Kubeflow.
-  - Note: Most of the logging APIs must be leveraged out of the box from either BPG or Spark - but we need to document, show case examples to user.
-
-Mentors expecting GSoC project execution plan - phase wise with weekly milestone that student want to achieve.
-
-#### Expected Outcomes:
-- A robust integration of BPG into the Kubeflow ecosystem, enabling efficient Spark job management across multiple clusters.
-- A Python SDK compatible with Kubeflow notebooks to facilitate Spark job submission through BPG, offering a scalable solution that reduces notebook load and enhances performance.
+  - Note: Most of the logging APIs must be leveraged out of the box from either BPG or Spark - but we need to document, showcase examples to user.
 - Comprehensive documentation and user guides to assist users in leveraging the new features effectively.
 
-#### Skills Required/Preferred:
-**Curious to learn, design, develop and having below skillsets will be plus : **
+__Skills Required/Preferred:__
 
 - Proficiency in Python, Java and familiarity with developing SDKs.
 - Experience with Kubernetes and managing containerized applications.
 - Understanding of Apache Spark and its deployment on Kubernetes clusters.
 - Familiarity with RESTful API development and integration.
 - Experience with monitoring tools and logging frameworks is a plus.
-#### Mentors:
-[Shekhar Prasad Rajak], [Software Development Engineer, Apple Inc], [Github: @Shekharrajak , Mail: shekharrajak@live.com]
-[Chaoran Yu], [Mentor Title/Position], [Github: @yuchaoran2011]
-[Andrey Velichkevich], [Mentor Title/Position], [Github @andreyvelich]
 
+---
+---
+
+### Project 7: GPU Testing for LLM Blueprints
 
-#### Difficulty:
-Medium to Hard
-#### GitHub Issue:
-Enhancing Kubeflow with Batch Processing Gateway for Efficient Multi-Cluster Spark Job Management spark-operator#2422
-#### Expected Project Size:
-350 hours
+__Components:__ Kubeflow Trainer (Training Operator)
 
+__Possible Mentors:__ [`@andreyvelich`](https://github.com/andreyvelich), [`@varodrig`](https://github.com/varodrig)
 
+__Difficulty:__ Medium
 
----
+__Size:__ 350 hours
 
-### Project 7
-### Title: Enable GPU Testing for LLM Blueprints in Kubeflow Trainer
-#### Component: Kubeflow Trainer
+__Goals:__
 
-#### Tracking issue: kubeflow/trainer#2432
-#### Component: Kubeflow Trainer
+- Explore using Self-Hosted Runners for GPU testing in Kubeflow Trainer ([`kubeflow/trainer#2432`](https://github.com/kubeflow/trainer/issues/2432))
 
-#### Skills required: 
-- GitHub Actions 
-- Kubernetes 
-- PyTorch 
+__Skills Required/Preferred:__
+
+- GitHub Actions
+- Kubernetes
+- PyTorch
 - Python
 
-#### Difficulty: hard
+---
+---
 
-#### Mentors: 
-- Andrey Velichkevich, 
-- Valentina Rodriguez Sosa
+### Project 8: Support JAX and TensorFlow Training Runtimes
```

Comment:
You can link these issues created by @Electronic-Waste 
- https://github.com/kubeflow/trainer/issues/2443
- https://github.com/kubeflow/trainer/issues/2442